src/main/java/frc/robot/subsystems/feeder/FeederIO.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.feeder;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface FeederIO {
```

Comment:
does this deliberately not have a comment?
sorry if I was not thorough enough, but suggestions last time were only duplicates wrt the whole PR.

Reply:
looking through the other PRs, you have deliberately removed these comments. OK, just making sure we are consistent.